dom0/sd-clean-all.sls

```diff
@@ -74,6 +74,11 @@ sd-cleanup-sys-firewall:
       - qvm-run sys-firewall 'sudo rm -f /etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation'
       - qvm-run sys-firewall 'sudo rm -f /etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation-test'
 
+sd-cleanup-whonix-gateway:
+  cmd.run:
+    - names:
+      - qvm-run whonix-gateway-17 'sudo apt purge --yes securedrop-keyring securedrop-qubesdb securedrop-whonix-config'
```

Comment:
This doesn't purge the apt-test file if it was created.

Reply:
Thanks for catching this.  I'm fine with the fix you've proposed in 3114359706310b306a2674fd4cb1a4456bd0f377 until we do what @rocodes proposes in <https://github.com/freedomofpress/securedrop-workstation/pull/1051#discussion_r1624796502>, about which I'll follow up there.